### ISSUE TITLE
imageSmoothingEnabled should not apply to stroke, fills

### ISSUE THREAD

**Issue by jacobp100** [joshmarinacci/node-pureimage#28]
It only applies to images [(spec)](https://html.spec.whatwg.org/multipage/canvas.html#image-smoothing). Should remove a few lines of code at least!

**Comment by joshmarinacci** [comment]
true. I left it in because doing chunky lines is occasionally useful. Instead of removing it, though, how about we change it to a custom property like. pureimageLineSmoothingEnabled?
